transform/mattermost-analytics/models/marts/marketing/fct_security_newsletter.sql

```diff
@@ -0,0 +1,38 @@
+with marketing as (
+    select marketing_id
+        , email as email
+        , server_id
+        , created_at
+        , updated_at
+    from
+        {{ ref('stg_cws__marketing')}}
+    where subscribed_content = 'security_newsletter'
+    qualify row_number() over (
+            partition by email
+            order by
+                created_at desc
+        ) = 1
+),
+security_newsletter as (
+    select distinct m.marketing_id as marketing_id
+        , m.email as email
+        , m.server_id as server_id
+        , l.lead_id as lead_id
+        , cm.campaign_member_id as campaign_member_id
+        , l.lead_id is not null as is_existing_lead
```

Comment:
```suggestion
        , l.lead_id is not null as is_existing_lead
        , '{{ var('marketing_newsletter_campaign_id') }}' as campaign_id
```

Campaign id should also be added here. This way it's going to be visible later in hightouch sync. Note that hightouch sync may also define it as a static value. However if there's a need for changing the campaign id, it will require making updates in two places.

Reply:
Added campaign id field.